self-api/src/main/java/com/selfxdsd/api/ApiTokens.java

```diff
@@ -0,0 +1,14 @@
+package com.selfxdsd.api;
+
+/**
+ * API Tokens.
+ */
+public interface ApiTokens extends Iterable<ApiToken> {
+    /**
+     * API tokens of a user.
+     *
+     * @param user User.
+     * @return ApiTokens.
+     */
+    ApiTokens ofUser(User user);
+}
```

Comment:
@andreoss Here as well, we need another todo, for the method ``ApiToken :: getById(String token)``.

Reply:
@amihaiemil Done.